schemas/telemetry/pioneer-study/pioneer-study.1.parquetmr.txt

```diff
@@ -0,0 +1,20 @@
+message pioneer-study {
+  required binary creationDate (UTF8);
+  required binary id (UTF8);
+  required binary type (UTF8);
+  required double version;
+  required group payload {
+    required binary encryptedData (UTF8);
```

Comment:
Yes, but right now we don't know the schema of what's going to be inside the encrypted blob for the first study. heatmap as such isn't going to launch at all. lonnen still wants to get the pipeline as such up and running quickly, so for a version 1 I decided to punt on the encryption part and let the first study consumer deal with it on its own. Using the common ping format also means we can re-use the client side telemetry code, which is another big win for a version 1 of this.